<launch>
	<arg name="node_name" default="pi_master_node"/>
	<group ns="/ubiquityrobot">
		<node name="$(arg node_name)" pkg="pi_driver" type="$(arg node_name).py" output="screen" ></node>
	</group>
</launch>
